boolean_algebra/quine_mc_cluskey.py

```diff
@@ -92,40 +92,34 @@ def selection(chart: list[list[int]], prime_implicants: list[str]) -> list[str]:
     temp = []
     select = [0] * len(chart)
     for i in range(len(chart[0])):
```

Comment:
`range(len())` is almost always a sign that `enumerate()` should have been used instead.  Can you please review all uses of `range(len())` to see which of them can be converted?

Reply:
All code is based on indices and indices are used for assignment (and for accessing elements of several lists). So, for me `evaluate` doesn't help much here

Reply:
hogwash.